.github/workflows/ci.yml

```diff
@@ -0,0 +1,46 @@
+name: Continuous Integration (CI)
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
```

Comment:
What do you think about having this CI pipeline running for all the created PRs?

Reply:
Agree, replacing the `branches` filter with this one you suggested:
```
on:
  pull_request:
    types: [opened, synchronize, reopened, ready_for_review]
```